src/elements/embed/Callout.tsx

```diff
@@ -0,0 +1,217 @@
+import { css } from "@emotion/react";
+import { neutral, space, text } from "@guardian/src-foundations";
+import { textSans } from "@guardian/src-foundations/typography";
+import React, { useEffect, useState } from "react";
+import { Error } from "../../editorial-source-components/Error";
+import { Label } from "../../editorial-source-components/Label";
+import { FieldLayoutVertical } from "../../editorial-source-components/VerticalFieldLayout";
+import type { FieldNameToValueMap } from "../../plugin/helpers/fieldView";
+import { unescapeHtml } from "../helpers/html";
+import { EmbedTestId } from "./EmbedForm";
+import type { createEmbedFields } from "./EmbedSpec";
+
+type Props = {
+  fieldValues: FieldNameToValueMap<ReturnType<typeof createEmbedFields>>;
```

Comment:
`nit`, we could simply pass it the HTML value as that's the only part of the `fieldValues` we need.

Reply:
I've done so now!